src/platform/atob-node.js

```diff
@@ -5,5 +5,10 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
-import atob from 'atob';
+//import atob from 'atob';
+
+function atob(str) {
```

Comment:
Do we need this change?

Reply:
This looks like we're replacing a trivial npm import with equivalent functionality, i.e. yes we should definitely be doing this :)